README.md

```diff
@@ -173,15 +164,13 @@ Setting up debugging in IntelliJ
 
 * In future debugging sessions, you can simply adjust the "Program Arguments" in the "Hellbender debug" configuration as needed
 
-Updating the Intellij project when dependencies change
--------------------
+##Updating the Intellij project when dependencies change
 If there are dependency changes in `build.gradle` it is necessary to refresh the gradle project. This is easily done with the following steps.
 
 * Open the gradle tool window  ( "View" -> "Tool Windows" -> "Gradle" )
 * Click the refresh button in the Gradle tool window.  It is in the top left of the gradle view and is represented by two blue arrows.
 
-Uploading Archives to Sonatype
--------------------
+##Uploading Archives to Sonatype
 To upload snapshots to sonatype you'll need the following:
 
 *You must have a registered account on the sonatype JIRA (and be approved as a hellbender uploader)
```

Comment:
I think you need to add a space after the asterisk to make it a bullet, otherwise it italicizes everything